kamerka.py

```diff
@@ -41,6 +42,9 @@
 parser.add_argument("--address", help="Address")
 parser.add_argument("--dark", help="Dark Theme", action='store_true')
 
+if len(sys.argv)==1:
+    parser.print_help(sys.stderr)
+    sys.exit(1)
```

Comment:
If one (or more) of the command-line arguments were mandatory, wouldn't this `if` be unnecessary?

Reply:
This is just in case of call it without arguments. Otherwise, none was displayed(at least here, on py2.7.15 @ OSX).

Reply:
> This is just in case of call it without arguments.

That's kind of the point of my comment, one (or more) arguments seem to be mandatory but that isn't reflected in the code. One way would be to add `required=True` to one (or more) arguments but it is somewhat frowned upon...

> Note: Required options are generally considered bad form because users expect options to be optional, and thus they should be avoided when possible.

The other way would be to use a positional argument instead. Something like this?

```python
parser.add_argument("address", help="Address")
```

Reply:
basically, --coordinates *OR* --address should be mandatory, I don't know if there is a flag on the lib that perform this kind of check, if there is any, make sense to use(please commit it). otherwise, I can't see any other way to do this

Reply:
Maybe something like `ArgumentParser.add_mutually_exclusive_group` is should only be able to be coordinates OR address, never both?

Reply:
yup, never both. And `add_mutually_exclusive_group` works for python2 and python3?

Reply:
```python
>>> import argparse
... 
... parser = argparse.ArgumentParser()
... 
... group = parser.add_mutually_exclusive_group(required=True)
... group.add_argument("--coordinates", help="Coordinates",
...                     default="")
... parser.add_argument("--radius", help="Radius in km (Default 3)", default="3")
... group.add_argument("--address", help="Address")
... parser.add_argument("--dark", help="Dark Theme", action='store_true')
... 
>>> parser.parse_args(['--coordinates', '123'])
Namespace(address=None, coordinates='123', dark=False, radius='3')
>>> parser.parse_args(['--address', '123'])
Namespace(address='123', coordinates='', dark=False, radius='3')
>>> parser.parse_args(['--address', '123', '--coordinates', '12345'])
usage: pydevconsole.py [-h] [--coordinates COORDINATES] [--radius RADIUS]
                       [--address ADDRESS] [--dark]
pydevconsole.py: error: argument --coordinates: not allowed with argument --address
```

Reply:
yup, it works (but as you stated, requires -- instead of a single -) here but the error msg is kinda weird/ugly. Do you know if we could append `parser.print_help(sys.stderr)` to the error msg generated by `parser.add_mutually_exclusive_group` ?
Also, considering this -, -- handling, this if would not be enough(although not idiomatic)?